packages/slice-machine/pages/index.tsx

```diff
@@ -1,122 +1,63 @@
 import EmptyState from "components/EmptyState";
-import { CustomType, ObjectTabs } from "@lib/models/common/CustomType";
 import useSliceMachineActions from "@src/modules/useSliceMachineActions";
 import { selectLocalCustomTypes } from "@src/modules/customTypes";
 import { useSelector } from "react-redux";
 import { SliceMachineStoreType } from "@src/redux/type";
 import { isLoading } from "@src/modules/loading";
 import { LoadingKeysEnum } from "@src/modules/loading/types";
+import { MdSpaceDashboard } from "react-icons/md";
+import { CustomType, ObjectTabs } from "@models/common/CustomType";
 
-// To isolate later
-const CTName: React.FunctionComponent<{ ctName: string }> = ({ ctName }) => {
-  return (
-    <Heading sx={{ flex: 1, lineHeight: 20 }} as="h6">
-      {ctName}
-    </Heading>
-  );
-};
-
-// To isolate later
-const CTRepeatble: React.FunctionComponent<{ repeatable: boolean }> = ({
-  repeatable,
-}) => {
-  return (
-    <Text sx={{ fontSize: 0, color: "textClear", lineHeight: "20px" }}>
-      {repeatable ? "Repeatable" : "Single"} Type
-    </Text>
-  );
-};
-
-// To isolate later
-const CTThumbnail = ({
-  heightInPx,
-  preview = null,
-  withShadow = true,
-}: {
-  heightInPx: string;
-  preview: { url: string } | null;
-  withShadow?: boolean;
-}) => {
+const CustomTypeTable: React.FC<{
+  customTypes: ReadonlyArray<CustomType<ObjectTabs>>;
+}> = ({ customTypes }) => {
   return (
     <Box
+      as={"table"}
       sx={{
-        backgroundColor: "headSection",
-        backgroundRepeat: "repeat",
-        backgroundSize: "15px",
-        backgroundImage: "url(/pattern.png)",
-        height: heightInPx,
-        overflow: "hidden",
-        display: "flex",
-        alignItems: "center",
-        justifyContent: "center",
-        borderRadius: "6px",
-        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-        border: (t) => `1px solid ${t.colors?.borders}`,
-        boxShadow: withShadow ? "0px 8px 14px rgba(0, 0, 0, 0.1)" : "none",
+        mt: "36px",
       }}
     >
-      <Box
-        sx={{
-          width: "100%",
-          height: "100%",
-          backgroundSize: "contain",
-          backgroundPosition: "50%",
-          backgroundRepeat: "no-repeat",
-          // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-          backgroundImage: "url(" + `${preview?.url}` + ")",
-        }}
-      />
+      <thead>
+        <tr>
+          <Box as={"th"} sx={{ width: "35%" }}>
+            Name
+          </Box>
+          <Box as={"th"} sx={{ width: "50%" }}>
+            API ID
+          </Box>
+          <Box as={"th"} sx={{ width: "15%" }}>
+            Type
+          </Box>
+        </tr>
+      </thead>
+      <tbody>
+        {customTypes.map((customType) => (
+          <Link passHref href={`/cts/${customType.id}`} key={customType.id}>
+            <tr key={customType.id}>
+              <Box as={"td"} style={{ width: "35%" }}>
```

Comment:
OPT: extract the box width so that they are being kept between the header and the value from the table.

Reply:
Can be a good idea, I will make the change

Reply:
Done